dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -79,18 +79,33 @@ Future<void> main() async {
         'App',
       ));
 
-      final String aotSymbols = await dylibSymbols(path.join(
+      final String appFrameworkPath = path.join(
         outputPath,
         'Debug',
         'App.framework',
         'App',
-      ));
+      );
+      final String aotSymbols = await dylibSymbols(appFrameworkPath);
 
       if (aotSymbols.contains('architecture') ||
           aotSymbols.contains('_kDartVmSnapshot')) {
         throw TaskResult.failure('Debug App.framework contains AOT');
       }
 
+      final String debugAppArchs = await fileType(appFrameworkPath);
+
+      if (!debugAppArchs.contains('armv7')) {
+        throw TaskResult.failure('Debug App.framework armv7 architecture missing');
+      }
+
+      if (!debugAppArchs.contains('arm64')) {
+        throw TaskResult.failure('Debug App.framework arm64 architecture missing');
+      }
+
+      if (!debugAppArchs.contains('x86_64')) {
+        throw TaskResult.failure('Debug App.framework x86_64 architecture missing');
```

Comment:
while you're here, we should check that profile and release has arm/64 and no x86/64 too

Reply:
It already checks armv7 and arm64 from the symbols table on line 126. I will add a check that it doesn't contain x86_64.